codegen/codegen_test.go

```diff
@@ -343,10 +354,21 @@ func TestCodeGen(t *testing.T) {
 		}
 		`,
 		func(t *testing.T, cg *CodeGen) solver.Request {
-			id, err := cg.LocalID(ctx, ".")
-			require.NoError(t, err)
-
-			return Expect(t, llb.Local(id, llb.SessionID(cg.SessionID()), llb.SharedKeyHint(".")))
+			return Expect(t, cg.Local(t, "."))
+		},
+	}, {
+		"local file",
+		[]string{"default"},
+		`
+		fs default() {
+			local "codegen_test.go"
```

Comment:
we probably want to add a explicit test to verify that include/exclude patterns are (and continue to be) ignored:
```
fs default() {
  local "codegen_test.go" with option {
    includePatterns "ignored"
    excludePatterns "ignored"
  }
}
```

Reply:
https://github.com/openllb/hlb/pull/141/commits/b8b8d5f7b4419bca9d27a7b3c3ec8c6656db2902